test/pdo_sqlsrv/pdo_336_pho_exec_empty_result_set_error.phpt

```diff
@@ -0,0 +1,53 @@
+--TEST--
+GitHub issue #336 - PDO::exec should not return an error with query returning SQL_NO_DATA
+--DESCRIPTION--
+Verifies GitHub issue 336 is fixed, PDO::exec on query returning SQL_NO_DATA will not give an error
+--SKIPIF--
+--FILE--
+<?php
+require_once("pdo_tools.inc");
+
```

Comment:
Looks like you don't have to include "pdo_tools.inc", but since you have it, you might want to create temporary tables instead (no need to check for a table's existence then drop it later)